vos/vos/commands/vtag.py

```diff
@@ -131,32 +134,50 @@ def vtag():
             vospace_certfile=opt.certfile,
             vospace_token=opt.token,
             insecure=opt.insecure)
-        node = client.get_node(node)
+        node = client.get_node(node_arg)
         if len(props) == 0:
             # print all properties
             pprint.pprint(node.props)
-
-        changed = False
-        for prop in props:
-            prop = prop.split('=')
-            if len(prop) == 1:
-                # get one property
-                pprint.pprint(node.props.get(prop[0], None))
-            elif len(prop) == 2:
-                # update properties
-                key, value = prop
+        if args.recursive:
```

Comment:
This is a very picky comment - using `args.recursive` and `opt.recursive` in the same set of `if` statements is inconsistent.

Reply:
That was indeed silly since both of them were pointing to the same object. Removed `opt`.